### PR TITLE
Add zrender@3.2.1 w/ git autoupdate.

### LINKED ISSUE #9229: [Request] Add zrender
**Library name:** zrender
**Git repository url:** https://github.com/ecomfe/zrender

### PR DESCRIPTION
**Pull request by Piicksarn**
@cdnjs/intern @cdnjs/intern2, 
I need your help for reviewing the pr. And the files are added by following :point_right: https://github.com/ecomfe/zrender/issues/215
**Pull request for issue: #9229**
Thank you all!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ecomfe/zrender
 * Official website (optional, not the repository): - 
 * NPM package url (optional): https://www.npmjs.com/package/zrender
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1603
   - Count of watchers: 185
   - Count of forks: 596
 * NPM download stats (optional):
   - Downloads in the last day: 357
   - Downloads in the last week: 2092
   - Downloads in the last month: 8554

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/zrender/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  "license": "BSD-2-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ecomfe/zrender.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "zrender*"
```
Comment (pvnr0082t) [inline ajax/libs/zrender/package.json]: Let's use `"**/zrender*"` here.

**Comment by maruilian11** [comment]
please add `homepage` https://ecomfe.github.io/zrender/index.html

**Comment by x09326** [comment]
The file structure in v1.0.0~v1.0.2 are different from other versions.
I think you need to add them manually.

File: ajax/libs/zrender/package.json (hunk 1 above)
Comment (Piicksarn) [inline ajax/libs/zrender/package.json]: @pvnr0082t, thank you!

**Comment by Piicksarn** [comment]
@cdnjs/intern @cdnjs/intern2,
I have it modified, and need ur help for reviewing.
Thank you all! 

Modification including:
 - modify package.json : **`homepage config`** and **`auto-update config`**
 - Add old versions which are different file structure.

**Comment by Piicksarn** [comment]
Dear all,
I modified the files be included.
Removing the .map files for making file structure consistent.
please help me for reviewing, thank u all.
@cdnjs/intern @cdnjs/intern2

**Comment by PeterDaveHello** [comment]
Ping @x09326 @kennynaoh

**Comment by x09326** [comment]
@Piicksarn Sorry for the late response, please rebase with the latest master branch, thanks.

**Comment by maruilian11** [comment]
@Piicksarn Please update to 3.2.2, thanks!

**Comment by PeterDaveHello** [comment]
Let's just save the time, if new version can be auto-updated, just let the bot do it, make sure the corrent things work.

**Comment by maruilian11** [comment]
@Piicksarn Let's not update to 3.2.2. Please mention the manually added version sources.

**Comment by Piicksarn** [comment]
@maruilian11, @x09326, @pvnr0082t,
dear all, i have it rebase to master and need ur help for checking.
thank you!

**Comment by maruilian11** [comment]
you didn't mention the manually added versions source in commit msg...

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Pull request closed without merging**